meson.build

```diff
@@ -0,0 +1,138 @@
+project('scs', 'c')
+
+py = import('python').find_installation(pure: false)
+
+cc = meson.get_compiler('c')
+blas_deps = []
+if host_machine.system() == 'darwin'
+  blas_deps = [dependency('Accelerate')]
+else
+  blas_deps = [cc.find_library('openblas', static: get_option('link_blas_statically'), required : false)]
+endif
+if not blas_deps[0].found()
+    blas_deps = [cc.find_library('blas', static: get_option('link_blas_statically'), required : false)]
+    cblas_dep = cc.find_library('cblas', static: get_option('link_blas_statically'), required : false)
+    if cblas_dep.found()
+        blas_deps += cblas_dep
+    endif
+endif
+
+# We find anaconda blas on windows for github CI
+if not blas_deps[0].found()
```

Comment:
This is specific to the CI setup for building Windows wheels, and won't work in case users attempt to build from source on Windows for some reason. That may or may not matter to you, but it's probably appropriate to at least emit a warning with `message("explain-that-blas-wasn't-found-and-this-is-for-CI-only")` here?

Also, you may want to change this line to
```meson
if host_machine.system() == 'windows' and not blas_deps[0].found()
    ....

else
    error('OpenBLAS or Netlib BLAS/CBLAS is required on all platforms except Windows, and was not found.')
```

Reply:
OK, I'm thinking of refactoring it: passing anaconda lib folder location as a meson option in the CI file, so the `meson.build` is not tailored for the compilation process on github.

Reply:
That makes sense to me. You can do it as a CLI option, or you can craft an `openblas.pc` file which points to the right paths in a CI job and then add the directory of that `.pc` file to `PKG_CONFIG_PATH`. For the latter approach, see http://scipy.github.io/devdocs/building/blas_lapack.html#using-pkg-config-to-detect-libraries-in-a-nonstandard-location.